sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/persistence/PersistenceAndroidTest.java

```diff
@@ -51,7 +51,7 @@ public void missingLogSerializer() throws Persistence.PersistenceException {
         //noinspection TryFinallyCanBeTryWithResources (try with resources statement is API >= 19)
         try {
             /* Generate a log and persist. */
```

Comment:
Can you add an empty line above as part of this PR if you need to add more commits?

Reply:
Between 55 and 56, too.